lib/web_ui/test/engine/recording_canvas_test.dart

```diff
@@ -37,21 +37,21 @@ void main() {
       underTest.drawDRRect(rrect, rrect.inflate(1), somePaint);
       underTest.apply(mockCanvas);
       // Expect nothing to be called
-      expect(mockCanvas.methodCallLog.length, equals(0));
+      expect(mockCanvas.methodCallLog.length, equals(1));
```

Comment:
This can mask a bug that this test is trying to avoid. For example, this test will pass if we do not call `endOfPaint` but call `drawDRRect`. We can filter out `endOfPaint` and check that it's 0, or check that the `methodCallLog.single.methodName` is `endOfPaint`.

Reply:
Done.